bylaws/2-cf-membership.md

```diff
@@ -30,13 +30,20 @@ A **project** within the CF represents collaborative innovation in open-source t
 
 **Dedicated Representation:** If an EGC representative is also elected to the [CF Council (CFC)][cfc], we recommend appointing an additional representative for the EGC. This helps maintain a focused approach to both specific project needs and the broader objectives of the CF.
 
+> [!NOTE]
+> An EGC member does not have to be a general CF member.
+> They are appointed by a project to represent its interests in [Foundation governance][governance].
```

Comment:
"They" -> "EGC members", since "they" is ambiguous here.

Reply:
I smashed this note into the paragraph above. It eliminates the vague "they" .. I believe it works better.